lib/vssolution.cpp

```diff
@@ -1908,6 +1952,7 @@ void VisualizationSceneSolution::PrepareNumbering()
 {
    PrepareElementNumbering();
    PrepareVertexNumbering();
+   PrepareFaceNumbering();
```

Comment:
This seems out-of-order for me, pick either `vertex-edge/face-element` or `element-edge/face-vertex`